ui/main.qml

```diff
@@ -138,6 +138,7 @@ ApplicationWindow {
         property var chatSplitView
         property var walletSplitView
         property var profileSplitView
+        property bool showBackupMessage: true
```

Comment:
Might make sense to call it `removeMnemonicAfterLogin` instead to make more clear what this is about?

Reply:
Yeah I think that's right. Also, it doesn't have to be a Settings property, since we don't care about saving in on multiple app reloads; the mnemonic only needs to be removed once.

So, you could probably just do a normal property called `removeMnemonicAfterLogin`, then you probably don't need all those other checks on `showBackupMessage` since the mnemonic being removed already takes care of it I think?

Reply:
> So, you could probably just do a normal property called removeMnemonicAfterLogin,

I've update the PR accordingly, just wondering what you referring to with "normal property". Do you mean a normal prop on `appMain`?

The reason I put it originally on `Settings` was because I was approaching a client-side only solution so, indeed I wanted to keep the value around. While this is not needed anymore, I still need access to that property in different places in the app. This could also be done by putting it on `appMain` but I thought it's prolly better to rely on `appSettings` since we use that throughout the app anyways.

Thoughts?